torch/distributed/rpc.py

```diff
@@ -73,7 +74,7 @@ def _init_rpc(backend=RpcBackend.PROCESS_GROUP,
             raise RuntimeError("self_rank argument {} doesn't match pg rank {}".format(
                                self_rank, group.rank()))
         # TODO: add try-except and destroy _agent in all processes if any fails.
-        _agent = ProcessGroupAgent(self_name, group)
+        _agent = ProcessGroupAgent(self_name, group, num_send_recv_threads)
         init_rref_context(_agent)
     elif is_backend_registered(rpc_backend):
```

Comment:
where is rpc_backend from? should it not be "backend"?

Reply:
@zhaojuanmao  Thanks. That's in @satgera 's PR. I am keeping each PR atomic.